pkg/ts/utils/log.ts

```diff
@@ -17,18 +19,15 @@ const _logFactory = (enable, preamble, color, log = 'log') => {
   return console[log].bind(console, `%c${preamble}`, style);
 };
 
-const logKinds =  ['log', 'group', 'groupCollapsed', 'groupEnd', 'dir'];
-const errKinds =  ['warn', 'error'];
-
-export const logFactory = (enable, preamble: string, color: string = '') => {
-  const loggers = {};
-  logKinds.forEach(log => loggers[log] = _logFactory(enable, `${preamble}`, color, log));
-  errKinds.forEach(log => loggers[log] = _logFactory(true, `${preamble}`, color, log));
-  // `log` is default
-  const log = loggers['log'];
-  // other loggers invoked by name
+export const logFactory = (enable: boolean, preamble: string, color: string = ''): Logger => {
+  const loggers: AllLoggerFunctions = {
+    ...(Object.fromEntries(logKinds.map(log => [log, _logFactory(enable, `${preamble}`, color, log)])) as DebugLoggers),
```

Comment:
`fromEntries` should be aliased off of Object above, there's a pattern in these modules for doing that.